manifests/init.pp

```diff
@@ -0,0 +1,69 @@
+# Class: lastpass
+#
+# This class installs the LastPass CLI and provides functions to interact with it.
+
+class lastpass (
+  $package  = $lastpass::params::package,
+  $home     = '/root/.lpass',
+  $agent_timeout = 3600,
+  $username = undef,
+  $password = undef,
+) inherits lastpass::params {
+
+  package { $package:
+    ensure => present,
+  }
+
+  file { '/usr/local/bin/lpasspw':
+    ensure => file,
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0755',
+    source => "puppet:///modules/${module_name}/lpasspw",
+  }
+
+  file { $home:
+    ensure => directory,
+    mode   => '0600',
+  }
+  file { "${home}/pw":
+    ensure  => file,
+    mode    => '0400',
+    content => inline_template('<%= @password %>'),
+  }
+
+  profiled::script { 'lpass.sh':
```

Comment:
After thinking about this some, I believe we need to split this up into two parts. The first will manage an automated user like it's doing now. It will need a manage_user flag and a manage_lp_home or something. It will do exactly what it's doing now, but only for that one user.

The second part will then set up LastPass in a way that other users can use it. It will not manage a password in that case.

I have to spend some more time thinking about how to implement both.

It seems to be that the first use is most important to us right now. As long as we are not creating a problem for ourselves, should we proceed with that?